### DISCUSSION TITLE
Heartbroken about (calssic) project migration

### DISCUSSION THREAD

**Discussion by AL0YSI0US** [community/community#125733] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I am only just now learning that classic projects will be retired and all previous projects will be automagically migrated to the new projects. 

Ahhhhhhhhhh I've been using these project boards to manage projects locally and globally for the past few years. The new boards do not allow me to organize information in a way that is accessible. 

I have tried darn near every web tool for organization and classic boards give me the most control on how to visualize data. 

I'm gonna share a link to the restoration project I'm working on. Without access to all this information in a place I can keep it organized I'm not sure what I'm going to do in the future. 

[The Heron's Nest LandBack Project](https://github.com/users/AL0YSI0US/projects/17#column-19309596)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by azenMatt** [comment, marked as answer]
Hi @AL0YSI0US - I'm really sorry to hear that. From looking at your project would I be right in saying it is the ability to just have long form content show on the cards rather than being behind a click that is key for you here? 

This isn't something we are looking to move over for the new projects but I appreciate you reaching out and hope you will be able to find something that will help keep you organised ❤️
